src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -567,20 +567,11 @@ def launch_fluent(
                 kwargs.update(cwd=cwd)
             if topy:
                 if isinstance(topy, str):
-                    journal_name = topy.split(".")[0]
-                    journal_ext = topy.split(".")[1]
-                    launch_string += f' -i {journal_name}.{journal_ext} -command="(api-start-python-journal \\\"\\\"{journal_name}.py\\\"\\\")"'  # noqa: E501
-                if isinstance(topy, list):
-                    all_scm_journal_names = ""
-                    all_py_journal_names = ""
-                    name_list = []
-                    for journals in topy:
-                        journal_name = journals.rsplit(".")[0]
-                        journal_ext = journals.rsplit(".")[1]
-                        name_list.append(journal_name)
-                        all_scm_journal_names += f' -i {journal_name}.{journal_ext}'
-                    all_py_journal_names += '_'.join(name_list)
-                    launch_string += f'{all_scm_journal_names} -command="(api-start-python-journal \\\"\\\"{all_py_journal_names}.py\\\"\\\")"'  # noqa: E501
+                    topy = [topy]
+                elif isinstance(topy, list):
```

Comment:
This should be an if, otherwise the code won't work when the passed topy is a string.